mlflow/sklearn.py

```diff
@@ -500,3 +506,184 @@ def should_log(self):
         return (self._parent is None) or (
             self._parent.allow_children and self._parent.clazz != self.clazz
         )
+
+
+def _is_old_version():
+    from sklearn import __version__
+
+    return LooseVersion(__version__) < LooseVersion("0.20.3")
+
+
+def all_estimators(type_filter=None):
```

Comment:
maybe we should create a directory named `sklearn` and seprate code into multiple files otherwise `mlflow/sklearn.py` would become very large and harder to maintain.